Ginger/Ginger/ALM/ALMDefectsProfilesPage.xaml.cs

```diff
@@ -187,22 +195,15 @@ private void AddDefectsProfile(object sender, RoutedEventArgs e)
             }
             newALMDefectProfile.Name = "Some Name " + (newALMDefectProfile.ID + 1).ToString();
             newALMDefectProfile.Description = "Some Description " + (newALMDefectProfile.ID + 1).ToString();
+            ALMConfig AlmConfig = ALMIntegration.Instance.GetDefaultAlmConfig();
+            newALMDefectProfile.AlmType = AlmConfig.AlmType;
             mALMDefectProfiles.Add(newALMDefectProfile);
 
             // mALMDefectProfileFields.ToList().ForEach(x => newALMDefectProfile.ALMDefectProfileFields.Add((ExternalItemFieldBase)x.CreateCopy()));
 
-            mALMDefectProfileFieldsExisted = new ObservableList<ExternalItemFieldBase>();
-            foreach (ExternalItemFieldBase aLMDefectProfileField in mALMDefectProfileFields)
-            {
-                ExternalItemFieldBase aLMDefectProfileFieldExisted = (ExternalItemFieldBase)aLMDefectProfileField.CreateCopy();
-                if (!string.IsNullOrEmpty(aLMDefectProfileField.ExternalID))
-                {
-                    aLMDefectProfileFieldExisted.ExternalID = string.Copy(aLMDefectProfileField.ExternalID);
-                }
-                aLMDefectProfileFieldExisted.PossibleValues = aLMDefectProfileField.PossibleValues;
-                mALMDefectProfileFieldsExisted.Add(aLMDefectProfileFieldExisted);
-            }
-            newALMDefectProfile.ALMDefectProfileFields = mALMDefectProfileFieldsExisted;
+            ALMIntegration.Instance.UpdateALMType(newALMDefectProfile.AlmType);
+            newALMDefectProfile.ALMDefectProfileFields = ALMIntegration.Instance.GetALMItemFieldsREST(true, ALM_Common.DataContracts.ResourceType.DEFECT, null);
+            newALMDefectProfile.ALMDefectProfileFields.Where(z => z.Mandatory != true).ToList().ForEach(x => x.SelectedValue = string.Empty);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)